backend/src/domain/error.ts

```diff
@@ -0,0 +1,20 @@
+export const DatabaseMigrationError = 'DATABASE_MIGRATION is optional, but if set must be one of [UP, DOWN]';
+export const DatabaseConnectionError = 'Missing environment variables for database connection';
+export const EntryNotFoundError = (tableName: string, filter: any) =>

```

Comment:
To align with your other naming standard suggestions I would rename the following database errors:
`EntryNotFoundError` to `DatabaseNotFoundError`
`InsertError` to `DatabaseInsertError`